tests/NLog.UnitTests/Targets/NetworkTargetTests.cs

```diff
@@ -907,7 +907,7 @@ internal class MySenderFactory : INetworkSenderFactory
             internal StringWriter Log = new StringWriter();
             private int idCounter;
 
-            public NetworkSender Create(string url, int maximumQueueSize, SslProtocols sslProtocols)
+            public NetworkSender Create(string url, int maxQueueSize, SslProtocols sslProtocols, int keepAliveTimeSeconds)
```

Comment:
proposal: change to TimeSpan and rename to keepAliveTime

Reply:
Easy to configure from XML ?

Reply:
Ahh good idea. Not part of the XML config. Will fix that

Reply:
> Easy to configure from XML

Yes, that will work but not for .NET 3.5 AFAIK

Reply:
See class LimitingWrapper and SetTimeSpanFromXmlTest

Reply:
Think TimeSpan-serialize-format is nice when days, hours, minutes, but not seconds.

Reply:
You enter 30, and you get 30 days.